.github/workflows/release.yml

```
@@ -242,12 +242,12 @@ jobs:
         uses: actions/checkout@v2
       - uses: actions/download-artifact@v2
         with:
-          name: moonbase-alphanet
-          path: build/alphanet
+          name: moonbeam
+          path: build
       - name: Prepare
         id: prep
         run: |
-          DOCKER_IMAGE=purestake/moonbase-parachain-testnet
+          DOCKER_IMAGE=purestake/moonbase-parachain
```

```suggestion
          DOCKER_IMAGE=purestake/moonbeam
```

Kind of makes sense for the image name to match the binary name.

---

This is the internal docker image. The public one is the next build step